pkg/docker/docker_command.go

```diff
@@ -97,23 +104,26 @@ func (c *DockerCommand) Inspect(image string) (*command.Manifest, error) {
 		"inspect",
 		image,
 	}
-	manifestData, err := c.exec("image", false, args...)
+	manifestData, err := c.exec("image", true, args...)
 	if err != nil {
 		return nil, err
 	}
 
 	decoder := json.NewDecoder(bytes.NewReader(([]byte(manifestData))))
-	var manifest command.Manifest
-	err = decoder.Decode(&manifest)
+	var manifests []command.Manifest
+	err = decoder.Decode(&manifests)
 	if err != nil {
 		return nil, err
 	}
 
-	return &manifest, nil
+	return &manifests[0], nil // Docker inspect returns us a list of manifests
```

Comment:
probably worth checking the `len(manifests)` and returning a non-nil `err` if it is empty rather than potentially panicking here